web/src/popover_menus.js

```diff
@@ -237,7 +237,6 @@ export function on_show_prep(instance) {
         e.stopPropagation();
         instance.hide();
     });
-    popovers.hide_all();
```

Comment:
So by using the `Enter` key to click an item inside the user card popover, I was able to create this state:

![image](https://github.com/zulip/zulip/assets/2746074/e0e26a5b-3f5f-4d98-991e-79181401f192)

Reply:
I didn't realize the hooks were not called for modals. Added a commit for that.